scalableminds/webknossos
# Filter datasets by organization, use that in task creation view

- Public datasets of other organizations are no longer listed during task creation. (Previously they were listed but still ultimately failed, compare #3568)
- Added new filter options for the dataset list api at `api/datasets`: `organizationName: String`, `onlyMyOrganization: Boolean`, `uploaderId: String`

### Steps to test:
- Set up two organizations, add datasets for both, set some datasets to public
- In task creation view you should not see other organization’s datasets in the dropdown menu, even if they are public
- try new filter options for dataset list: `organizationName: String`, `onlyMyOrganization: Boolean`, `uploaderId: String`

### Issues:
- fixes #5923
- fixes #5598
- fixes #6329
- related: #3568

------
